1-org/envs/shared/log_sinks.tf

```diff
@@ -82,14 +72,16 @@ module "logs_export" {
     Send logs to Logbucket
   *****************************************/
   logbucket_options = {
-    logging_sink_name   = "sk-c-logging-logbkt"
-    logging_sink_filter = local.all_logs_filter
-    name                = "logbkt-org-logs-${random_string.suffix.result}"
-    location            = local.default_region
+    logging_sink_name          = "sk-c-logging-logbkt"
```

Comment:
why is the sink name hardcoded.

Reply:
The terraform example foundation is an example of how to compose the modules in  [terraform-google-modules](https://github.com/terraform-google-modules) to create a secure foundation.

Most of the Terraform resources, projects, buckets, sinks, ..., created have hardcoded names that comply with the naming conventions defined in the documentation. The naming of the resources is also part of the example.

Some resources have a random suffix added if they use a global namespace like projects.